src/generator/operations.ts

```diff
@@ -253,18 +253,53 @@ function generateOperation(clientName: string, op: Operation, imports: ImportMan
   }
   const info = <OperationNaming>op.language.go!;
   const params = getAPIParametersSig(op, imports);
-  const returns = genereateReturnsInfo(op, false);
+  const returns = generateReturnsInfo(op, false);
   let text = '';
   if (hasDescription(op.language.go!)) {
     text += `// ${op.language.go!.name} - ${op.language.go!.description} \n`;
   }
-  text += `func (client *${clientName}) ${op.language.go!.name}(${params}) (${returns.join(', ')}) {\n`;
   // split param list into individual params
   const reqParams = getCreateRequestParametersSig(op).split(',');
   // slice off the parameter names from the type/type tuples
   for (let i = 0; i < reqParams.length; ++i) {
     reqParams[i] = reqParams[i].trim().split(' ')[0];
   }
+  // TODO Exception for Pageable LRO operations NYI
+  if (isLROOperation(op)) {
+    text += `func (client *${clientName}) Begin${op.language.go!.name}(${params}) (${returns.join(', ')}) {\n`;
```

Comment:
Move adding Begin prefix to namer.

Reply:
Well....I thought about this when I was implementing it. I can do it, but I thought it would be better NOT to rename the operation to use begin. Since the original operation name is used/referred to in more places than the name with the Begin prefix which is only used in the interface and method declarations. What do you think?

Reply:
Hmm this is a good point, it would spill into response envelope type names which is probably not what we want.  I wonder though if we should do this in the namer via something like `op.language.go!.methodPrefix` which we'd only set for LROs (although conceivably it could be used in the future for other things).  Then during interface/method generation if `methodPrefix` exists we add it.

Reply:
Yes, I'll do that. What other future uses do you think it will be useful for? So that I keep that in mind when implementing this.

Reply:
It could be useful if we want to add a `List` prefix to pageable methods.

Reply:
Ok sounds good, I added a methodPrefix